plugins/generator-1.19.2/forge-1.19.2/mappings/screens.yaml

```diff
@@ -75,4 +75,17 @@ ShulkerBox: ShulkerBoxScreen
 SmithingTable: SmithingScreen
 Smoker: SmokerScreen
 Stonecutter: StonecutterScreen
-KeyBinds: KeyBindsScreen
\ No newline at end of file
+KeyBinds: KeyBindsScreen
+AccessibilityOptions: AccessibilityOptionsScreen
+BanNotice: BanNoticeScreen

```

Comment:
BanNoticeScreen is not even a Screen but just a utility class. Please verify the objects you added to the lists.

Reply:
✔️ Fixed